recipes/enso_metrics/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "enso_metrics" %}
+{% set version = "v1.1.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/CLIVAR-PRP/ENSO_metrics/archive/refs/tags/{{ version }}.tar.gz
+    sha256: e933f373be1d793b801ed7155690fc4759a463da5931f18b5117ba9b259840b3    
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - cdms2
+    - cdutil
+    - genutil
+    - numpy
+    - scipy
+    - udunits2
```

Comment:
This is not listed upstream, can you confirm that this is correct?

Reply:
cdms2: https://anaconda.org/conda-forge/cdms2
genutil: https://anaconda.org/conda-forge/genutil
cdutil: https://anaconda.org/conda-forge/cdutil
numpy: https://anaconda.org/conda-forge/numpy
scipy: https://anaconda.org/conda-forge/scipy
udunits2: https://anaconda.org/conda-forge/udunits2

Reply:
Is there a specific key to use for this? "install_requires" doesn't work because a lot of these packages don't have a PyPI version.

Reply:
Ahh ok no that's fine then. So the upstream dependencies are not correct then?

Reply:
@BastianZim Yes that is correct. I got this list from lee1043 who is a contributor to the upstream project.

Reply:
Ok that's fine then.